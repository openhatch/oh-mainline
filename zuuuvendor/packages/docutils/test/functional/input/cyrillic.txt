Заголовок
---------

первый пример: "Здравствуй, мир!"

Title
-----

.. class:: language-en

first example: "Hello world".

Notes
-----

.. class:: language-en

This example tests rendering of Latin and Cyrillic characters by the LaTeX
and XeTeX writers. Check the compiled PDF for garbage characters in text and
bookmarks.

.. class:: language-en

To work around a problem with Cyrillic in PDF-bookmarks in `hyperref`
versions older than v6.79g 2009/11/20, the test caller ``latex_cyrillic.py``
sets ``hyperref_options`` to ``'unicode=true'`` while ``xetex_cyrillic.py``
sets it to ``'unicode=false'``. The recommended option for current
(2011-08-24) hyperref versions is ``'pdfencoding=auto'``.




