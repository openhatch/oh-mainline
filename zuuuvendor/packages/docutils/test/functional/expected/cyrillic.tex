\documentclass[a4paper,russian]{article}
% generated by Docutils <http://docutils.sourceforge.net/>
\usepackage{fixltx2e} % LaTeX patches, \textsubscript
\usepackage{cmap} % fix search and cut-and-paste in Acrobat
\usepackage{ifthen}
\usepackage[T1,T2A]{fontenc}
\usepackage[utf8]{inputenc}
\usepackage[english,russian]{babel}
\setcounter{secnumdepth}{0}

%%% Custom LaTeX preamble
% PDF Standard Fonts
\usepackage{mathptmx} % Times
\usepackage[scaled=.90]{helvet}
\usepackage{courier}

%%% User specified packages and stylesheets
\usepackage{cmlgc}

%%% Fallback definitions for Docutils-specific commands

% titlereference role
\providecommand*{\DUroletitlereference}[1]{\textsl{#1}}

% hyperlinks:
\ifthenelse{\isundefined{\hypersetup}}{
  \usepackage[colorlinks=true,linkcolor=blue,urlcolor=blue,unicode=true]{hyperref}
  \urlstyle{same} % normal text font (alternatives: tt, rm, sf)
}{}


%%% Body
\begin{document}


\section{Заголовок%
  \label{id1}%
}

первый пример: «Здравствуй, мир!»


\section{Title%
  \label{title}%
}

\foreignlanguage{english}{first example: “Hello world”.}


\section{Notes%
  \label{notes}%
}

\foreignlanguage{english}{This example tests rendering of Latin and Cyrillic characters by the LaTeX
and XeTeX writers. Check the compiled PDF for garbage characters in text and
bookmarks.}

\foreignlanguage{english}{To work around a problem with Cyrillic in PDF-bookmarks in \DUroletitlereference{hyperref}
versions older than v6.79g 2009/11/20, the test caller \texttt{latex\_cyrillic.py}
sets \texttt{hyperref\_options} to \texttt{'unicode=true'} while \texttt{xetex\_cyrillic.py}
sets it to \texttt{'unicode=false'}. The recommended option for current
(2011-08-24) hyperref versions is \texttt{'pdfencoding=auto'}.}

\end{document}
